Userland/Services/DHCPClient/DHCPv4.h

```diff
@@ -123,7 +123,9 @@ struct ParsedDHCPv4Options {
         auto& value = option.value();
         if (value.length != sizeof(T))
             return {};
-        return *(const T*)value.value;
+        T t;
+        __builtin_memcpy(&t, value.value, value.length);
+        return t;
```

Comment:
Can we replace this with `bit_cast` for some idiomatic goodness?

Reply:
I don't think so? bit_cast takes a const U& and returns a T, but the value struct here is a void* and a size_t. Not sure how to massage that into a bit_cast without refactoring these "value" structs.